pinot-filesystem/src/main/java/com/linkedin/pinot/filesystem/LocalPinotFS.java

```diff
@@ -40,6 +40,11 @@ public LocalPinotFS() {
   public void init(Configuration configuration) {
   }
 
+  @Override
+  public void mkdir(URI uri) throws IOException {
```

Comment:
Also, noticed that most other api's return a boolean but this one is void, any reason for that? If not, lets try to be consistent.

Reply:
wanted to migrate all to void and throw exception always to be consistent across filesystems but can leave it as boolean for now